### PR TITLE
Add pstream-clear

### PR DESCRIPTION
**Pull request by benoid**
Would you mind having a look to see if I did this safely before merging?

Thanks!

### CODE REVIEW

File: rsound/sequencer.rkt [hunk 1]
```diff
@@ -263,6 +263,18 @@
   (check-equal? (heap-count playing-heap) 0)
   (check-equal? (clear-ended-sounds! playing-heap 40) #f))
 
+(let ()
```
Comment (lehitoskin) [inline rsound/sequencer.rkt]: Shouldn't this be
```racket
(module+ test
  ...)
```

Comment (benoid) [inline rsound/sequencer.rkt]: I think all of the tests are added to the ```test``` module at the end of the file.  Thanks for the feedback though!

File: rsound/sequencer.rkt [hunk 2]
```diff
@@ -248,6 +263,18 @@
...
+  (define playing-heap (make-heap (lambda (a b)
```
Comment (jbclements) [inline rsound/sequencer.rkt]: I think just calling `make-unplayed-heap` would be better here, no?

File: rsound/stream-play.rkt [hunk 3]
```diff
@@ -42,6 +43,12 @@
     (raise-argument-error 'pstream-current-frame "pstream" 0 pstream))
   ((pstream-time-checker pstream)))
 
+;; Clear all queued rsounds in a pstream.  
+;; This does not stop rsounds which are currently
+;; being played.
+(define (pstream-clear pstream)
```
Comment (jbclements) [inline rsound/stream-play.rkt]: I think `pstream-clear` should be called `pstream-clear!`.

File: .gitignore [hunk 4]
```diff
@@ -1,4 +1,5 @@
 doc
+
```
Comment (jbclements) [inline .gitignore]: Is there a need for a blank line here?

**Comment by benoid** [comment]
Requested changes have been completed.  All tests are still passing.

Thanks for the feedback!  This is the first I've been made aware of commit squashing, and wow, is it cool.

**Comment by jbclements** [comment]
Looks good. Many thanks for making changes.

**Pull request merged**